### ISSUE TITLE
Arch linux, KDE 6, clipboard and Wayland problem

### ISSUE THREAD

**Issue by loi777** [flameshot-org/flameshot#3551]
### Flameshot Version

Flameshot v 12.1.0

### Installation Type

Linux, MacOS, or Windows Package manager (apt, pacman, eopkg, choco, brew, ...)

### Operating System type and version

Linux ver 6.8.1, with Arch and KDE 6

### Description

I used flameshot with KDE 5 and xorg normally, but ever since it updated to KDE 6 and wayland it totally is broken. I installed from KDE discover and for some strange reason that is the only way I can get it to work, the pacman AUR version doesn't even open.

My problem is that although I can print screen a rectangular in my screen and save it in any format, I can't copy it to my clipboard by pressing ctrl+c, like I used to do before.

I've seem that there might be a way to change the config file to understand it is wayland but I missed a proper tutorial to do so.

Also a minor problem that I already fixed, is that the printscreen button seems to have different key code in wayland, so I needed to manually change the key recognition in my KDE to open flameshot.

### Steps to reproduce

1. run in KDE 6 Wayland
2. print screen a region of the screen and copy it with ctrl+c.

### System Information

Linux, ver 6.8.1
16:9 1920x1080, 60hz
linux Arch (dunno the exact version but pretty up to date)
KDE Plasma 6, wayland

**Comment by omeringen** [comment]
> I installed from KDE discover and for some strange reason that is the only way I can get it to work, the pacman AUR version doesn't even open.

If you're using arch you should forget about discover. 
And AUR ? Flameshot at official repos just works fine(including ctrl+c) on up to date arch/plasma.

**Comment by loi777** [comment]
I did a work around, but forgot to update here.

I ended up deleting the flameshot from discover, and downloaded the one from the AUR.

My KDE still doesn't recognize it at all, but I noticed that using flameshot via bash works. So I just set up some key bindings in KDE to automatically bring up flameshot from the bash when pressing the print screen.

I think what happened is that KDE still thinks the executable is the old version from discover, so it tries to open it but can't find anything.
